modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala

```diff
@@ -16,34 +17,38 @@ trait RedisSortedSetAsyncCommands[K, V] {
   /**
    * Remove and return the member with the lowest score from one or more sorted sets
    * @param timeout The timeout
+   * @param direction Which end of the sorted set to pop from, the min or max
    * @param keys The keys
-   * @return The member with the lowest score, or None if the sets are empty
+   * @return The member removed based on the pop direction
    */
-  def bzPopMin(timeout: Long, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
+  def bzMPop(timeout: FiniteDuration, direction: SortOrder, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
 
   /**
-   * Remove and return the member with the highest score from one or more sorted sets
+   * Remove and return up to count members from the end of one or more sorted sets based on the pop direction (min or max)
    * @param timeout The timeout
+   * @param count The number of members to pop
+   * @param direction Which end of the sorted set to pop from, the min or max
    * @param keys The keys
-   * @return The member with the highest score, or None if the sets are empty
+   * @return The members removed based on the pop direction
    */
-  def bzPopMin(timeout: Double, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
+  def bzMPop(timeout: FiniteDuration, count: Int, direction: SortOrder, keys: K*): Future[List[ScoreWithKeyValue[K,V]]]
 
   /**
-   * Remove and return the member with the highest score from one or more sorted sets
+   * Remove and return the member with the lowest score from one or more sorted sets
    * @param timeout The timeout
    * @param keys The keys
-   * @return The member with the highest score, or None if the sets are empty
+   * @return The member with the lowest score, or None if the sets are empty
    */
-  def bzPopMax(timeout: Long, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
+  def bzPopMin(timeout: FiniteDuration, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
```

Comment:
I switched all the sorted set commands from specifying timeouts for blocking operations from longs and doubles to FiniteDuration. much clearer about intent.